app/controllers/schools/add_participants_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Schools
+  class AddParticipantsController < ::Schools::BaseController
+    FORM_SESSION_KEY = :add_participant_form
+    FORM_PARAM_KEY = :schools_add_participant_form
+
+    skip_after_action :verify_authorized
+    before_action :set_school_cohort
+
+    helper_method :add_participant_form
```

Comment:
I wonder if it's nicer to do it before relevant actions. I don't like the idea of views creating @ variables shared with controller when they render. 

I don't think it does anything bad here though, just a code style preference.

Reply:
Hmmm, I think I'd m,uch rather prefer a helper method than sharing an instance variable between two separate objects. I am aware that's how rails work by default, but it feels, well, dirty - it's a result of adaptation of MVC pattern that has been created for a GUI applications and really doesn't fit web server architecture...